src/theory/sets/rels_utils.h

```diff
@@ -10,90 +10,60 @@
  * directory for licensing information.
  * ****************************************************************************
  *
- * Extension to Sets theory.
+ * Utility functions for relations.
  */
 
 #ifndef SRC_THEORY_SETS_RELS_UTILS_H_
 #define SRC_THEORY_SETS_RELS_UTILS_H_
 
-#include "expr/dtype.h"
-#include "expr/dtype_cons.h"
 #include "expr/node.h"
 
 namespace cvc5 {
 namespace theory {
 namespace sets {
 
-class RelsUtils {
+class RelsUtils
+{
+ public:
+  /**
+   * compute the transitive closure of a binary relation
+   * @param members constant nodes of type (Tuple E E) that are known to in the
+   * relation rel
+   * @param rel a binary relation of type (Set (Tuple E E))
+   * @pre all members need to be constants
+   * @return the transitive closure of the relation
+   */
+  static std::set<Node> computeTC(std::set<Node> members, Node rel);
```

Comment:
Thanks for updating this, let's update this to pass a reference. `std::set<Node>& members`.

Reply:
Done.